odc/geo/crs.py

```diff
@@ -122,8 +122,8 @@ def __init__(self, crs_spec: Any):
     def __getstate__(self):
         return {"crs_str": self._str}
 
-    def __setstate__(self, state):
-        self.__init__(state["crs_str"])
+    def __setstate__(self, state) -> None:
+        self.__init__(state["crs_str"])  # type: ignore[misc]
```

Comment:
I guess mypy wants us to do something like:

```suggestion
    def __setstate__(self, state) -> None:
        self._crs, self._str, self._epsg = _make_crs(state["crs_str"])
```

Reply:
Nice!